vpn/src/main/java/com/duckduckgo/mobile/android/vpn/apps/TrackingProtectionAppsRepository.kt

```diff
@@ -94,12 +98,13 @@ class RealTrackingProtectionAppsRepository @Inject constructor(
     override suspend fun getExclusionAppsList(): List<String> = withContext(dispatcherProvider.io()) {
         val exclusionList = appTrackerRepository.getAppExclusionList()
         val manualExclusionList = appTrackerRepository.getManualAppExclusionList()
-        return@withContext packageManager.getInstalledApplications(PackageManager.GET_META_DATA)
+        val exclusionAppsList = packageManager.getInstalledApplications(PackageManager.GET_META_DATA)
             .asSequence()
             .filter { shouldBeExcluded(it, exclusionList, manualExclusionList) }
             .sortedBy { it.name }
             .map { it.packageName }
             .toList()
+        return@withContext exclusionAppsList
```

Comment:
nit: We don't need the `exclusionAppsList` intermediate variable just to return it later.

```suggestion
        return@withContext packageManager.getInstalledApplications(PackageManager.GET_META_DATA)
            .asSequence()
            .filter { shouldBeExcluded(it, exclusionList, manualExclusionList) }
            .sortedBy { it.name }
            .map { it.packageName }
            .toList()
```